lib/Doctrine/Common/DataFixtures/Executor/ORMExecutor.php

```diff
@@ -20,19 +20,23 @@ class ORMExecutor extends AbstractExecutor
     /** @var ORMReferenceListener */
     private $listener;
 
+    /** @var bool */
+    private $singleTransaction;
```

Comment:
Can I use `private bool $singleTransaction;`?

Reply:
Nope: this package still supports PHP 7.2, and type properties are a 7.4 feature.

Reply:
Right!